src/v/cluster/archival/ntp_archiver_service.cc

```diff
@@ -2328,17 +2328,16 @@ ntp_archiver::replicate_archival_metadata(
         auto is_validated = checks_disabled ? cluster::segment_validated::no
                                             : cluster::segment_validated::yes;
         cluster::emit_read_write_fence rw_fence = std::nullopt;
-        if (!fence.unsafe_add) {
+        if (fence.emit_rw_fence_cmd) {
             // The fence should be added first because it can only
             // affect commands which are following it in the same record
             // batch.
             vlog(
               archival_log.debug,
-              "add_segments, read-write fence: {}, unsafe add: {}, manifest "
+              "add_segments, read-write fence: {}, manifest "
               "last "
               "applied offset: {}, manifest in-sync offset: {}",
               fence.read_write_fence,
-              fence.unsafe_add,
```

Comment:
did you mean to do this? the commit does a rename, but this removes the state from the logging entirely?

Reply:
Yes, this code is not invoked if unsafe_add is false, so it will always be the same in the log line.